meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html

```diff
@@ -13,50 +13,64 @@
         <span>{% trans 'Create project' %}</span>
         {% endif %}
     </li>
+
+
     <li class="progress-line__entry">
         <span class="progress-line__line"></span>
         {% if project_progress.project_is_complete %}
             <i class="fas fa-check-circle u-success progress-line__icon" title="{% trans 'Project ready for publication' %}" aria-label="{% trans 'Project ready for publication' %}"></i>
         {% else %}
-            <i class="fas fa-circle progress-line__icon" title="{% trans 'Project not ready for publication' %}" aria-label="{% trans 'Project not ready for publication' %}"></i>
-        {% endif %}
-        {% if project.project_type == 'meinberlin_bplan.Bplan' %}
-        <span>{% trans 'Fill in development plan details' %}</span>
-        {% elif project.project_type == 'meinberlin_extprojects.ExternalProject' %}
-        <span>{% trans 'Fill in linkage details' %}</span>
-        {% elif project.project_type == 'meinberlin_projectcontainers.ProjectContainer' %}
-        <span>{% trans 'Fill in container details' %}</span>
-        {% else %}
-        <span>{% trans 'Fill in project details' %}</span>
+            <i class="fas fa-circle progress-line__icon--active" title="{% trans 'Project not ready for publication' %}" aria-label="{% trans 'Project not ready for publication' %}"></i>
         {% endif %}
+
+
+        <span class="{% for item in dashboard_menu.project %}{{ item.is_active|yesno:"progress-line__entry--active," }}{% endfor %}">
+            {% if project.project_type == 'meinberlin_bplan.Bplan' %}
```

Comment:
this is how I dealt with it?!

Reply:
I think that's a good idea! Will put it into a templatetag with the cleanup, but merge now!